recipe/meta.yaml

```diff
@@ -20,15 +19,17 @@ requirements:
     - pip
     - setuptools
     - wheel
+    - flit-core >=3.2,<4
+    # flit-core requires tomli
+    - tomli
```

Comment:
Yeah, `tomli` is inside `flit` package https://github.com/AnacondaRecipes/flit-feedstock/blob/ce786a1e1c5d2d65bd3ce5436a1ec6f45e96bb93/recipe/meta.yaml#L59

Reply:
`flit-core` was replaced by `flit`
https://github.com/AnacondaRecipes/pyparsing-feedstock/blob/update_to_3.0.9/recipe/meta.yaml#L22

Reply:
Does the build fail if you just use `flit-core` instead of `flit`?

Reply:
That is correct, that is why we are currently using `flit`

Reply:
What's the error message when we use `flit-core`? The conda-forge recipe builds fine with `flit-core` as the dependency.

Reply:
When we build it using `flit-core` there is a message stating that `tomli` is not present. That is why we now have a rebuild for  `flit` and `flit-core`:

Rebuild `flit` version `3.6.0`
https://github.com/AnacondaRecipes/flit-feedstock/pull/7

Reply:
We should not use `flit-core`
`tomli` and `tomli-w` have been removed from `flit-core` since version `3.6.0`
https://github.com/conda-forge/flit-feedstock/commit/a61caf2c0f1f84e1c8ef20de517f3c4192538923